src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java

```diff
@@ -186,7 +187,14 @@ public void vuapResolvedStateShouldHaveLinkToFandomArticleOnImage(Page page, Str
   public void vuapResolvedStateShouldCloseAfterTapingOnCloseButton(Page page, String slot, String videoIframeSelector) {
     final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
     final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+    // make sure slot is rendered so we get resolved state after the refresh
+    scrollToSlot(slot, ads);
+    vuap.replay();
+
     ads.refreshPage();
+    // TODO remove wait in ADEN-5481
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)